### PR TITLE
E-Commerce use case

### PR DESCRIPTION
**Pull request by bumblefudge**
Added to intro section

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -268,6 +272,39 @@ <h5>Oil and Gas Imports</h5>
...
+            data model is required, whereby many different actors can each
+            submit data points separately that get combined at time of customs
+            processing.
+          </p>
+          <p>
+            Without strong identification of legal entities (i.e.,
+            legally defined and registered supply chain actors) and of products,
+            and without high levels of semantic flexibility, the shards can be
+            quite hard to combine usefully. Linking together the registration of
+            individual packages with the pre-registration of commercial products
```
Comment (TallTed) [inline docs/index.html:294]:
```suggestion
            quite hard to combine usefully. Linking the registration of individual
            packages together with the pre-registration of commercial products
```

File: docs/index.html [hunk 2]
```diff
@@ -268,6 +272,39 @@ <h5>Oil and Gas Imports</h5>
...
+            and actors is the key value-add of this system, but could also be a
+            burdensome request on importers, retailers, and freight forwarders.
+            To minimize this burden, we are aligning wherever possible with the
+            ontology work of GS1 (GTINs and vLEIs), and with international
+            logistics consortia to align on shipping and tracking semantics
+            already adopted today. We distinguish between the VCs that are
```
Comment (TallTed) [inline docs/index.html:300]:
```suggestion
            ontology work of GS1 (GTINs and vLEIs), and with shipping and 
            tracking semantics already adopted today by international
            logistics consortia. We distinguish between the VCs that are
```

**Comment by bumblefudge** [comment]
thanks for the edits @TallTed !

File: docs/index.html [hunk 3]
```diff
@@ -233,7 +236,8 @@ <h5>Food and Agriculture</h5>
           </dd>
           <dt><dfn>FSMA Inspection</dfn></dt>
           <dd>
-            <a href="https://www.fda.gov/food/guidance-regulation-food-and-dietary-supplements/food-safety-modernization-act-fsma">Food
+            <a
```
Comment (mprorock) [inline docs/index.html:239]: any reason for the whitespace changes outside of the core e-commerce section?  looks like a formatter may have kicked in or something.  Mind backing out all changes except for the direct changes to your section on e-commerce?  at that point i will go ahead and merge

**Review by mprorock — CHANGES_REQUESTED** [review]

**Review by mprorock — APPROVED** [review]

**Pull request merged**